package.json

```diff
@@ -29,11 +31,13 @@
     "axios-mock-adapter": "^1.15.0",
     "babel-core": "7.0.0-bridge.0",
     "babel-jest": "^23.6.0",
+    "node-sass": "^4.9.0",
     "pug-lint-vue": "^0.1.3",
-    "stylus": "^0.54.5",
-    "stylus-loader": "^3.0.2",
+    "sass-loader": "^7.0.1",
+    "stylelint-config-standard": "^18.2.0",
     "vue-cli-plugin-pug": "^1.0.7",
-    "vue-template-compiler": "^2.5.16"
+    "vue-template-compiler": "^2.5.16",
+    "vuetify-scss": "^1.2.0"
```

Comment:
Is this library necessary? We import the vuetify CSS already built, and we don't want to have to know about the upstream build toolchain unless we really have to.